package.json

```diff
@@ -25,7 +25,7 @@
     "build:hugo:debug": "hugo -d ../dist -s site -v --templateMetrics --debug",
     "build:pdfjs": "./build-scripts/build-pdfjs.sh",
     "build:webpack": "cross-env NODE_ENV=production webpack --config src/webpack/webpack.prod.js",
-    "build:zips": "node ./build-scripts/build_course_zips.js -d dist -c site/content/courses -z site/static/zips",
+    "build:zips": "node ./build-scripts/build_course_zips.js -d dist -c site/content/courses -z zips",
```

Comment:
Can we make the argument passed to `-z` an environment variable and document it in `README.md`?  For example:

`-z $COURSE_ZIPS_DESTINATION`

Reply:
Should we have a default value here?

Reply:
I would say `zips` is a good default the way you have it, but we should do that in the script itself and still pass the env variable value here and check it, falling back to the default.